### ISSUE TITLE
Modify action link not shown if non default package name is used

### ISSUE THREAD

**Issue by friesoft** [Icinga/icingaweb2-module-director#1530]
## Expected Behavior
Modify action link on hosts and services should also be shown in case of non default package name (not "director")
Using two director databases, one with the default package name and the other one with a custom.

## Current Behavior
Modify Action link only shows for hosts/services using the "director" package name.

## Steps to Reproduce (for bugs)
Use a non standard "Icinga package name" (/icingaweb2/director/config/settings) and deploy your config. The action link doesn't show up.

## Your Environment
* Director version (System - About): master (c2d54f9de92de8f066625a680c40537aa481e0e7)
* Icinga Web 2 version and modules (System - About): 2.5.3
* Icinga 2 version (`icinga2 --version`): v2.8.4-745-g4eb5b09
* Operating System and version: Centos 7.4 (Icinga Vagrant Box Standalone)

**Comment by friesoft** [comment]
Hm.. it somehow seems to be the other way round (looks like I mixed it up). The host using the non standard package name shows the modify action link.

If director is currently switched to the other database with the default packagename I get the following error on clicking the link: Failed to load icinga_host "host"

Is this a separate bug? My expectation would be if my user has the permission for all director databases it should use the appropriate one. But the object would have to know which database it came from to generate an appropriate link switching director to this database..

**Comment by Thomas-Gelf** [comment]
Yep, this is a bug. This change was pretty intrusive and there are still some related issues I have to deal with :laughing:

**Comment by Thomas-Gelf** [comment]
@friesoft: I have to postpone this, sorry. Checking every Director DB when showing a single service in the monitoring module is not an option. I have currently 20-30 of them, but even with less DBs it would be too fragile and slow.

We have a change in the pipeline that will replace the current Job Daemon with generic daemon taking care of more background jobs. It will handle migrations, pre-render configs, of course deal with our Jobs - and it will know about all your instances and their current health. So this will then be one call to that daemon to ask for the DB fitting the given package, and then a single call to the correct DB to look it up.

I'll not risk to break 1.5 with this change, that's why I have to postpone this.

**Comment by Thomas-Gelf** [comment]
v1.7 will now ship the background daemon I mentioned, immediately after the release we can start to work on such features.
